docs/cli.txt

```diff
@@ -79,6 +82,25 @@ Example Usage with SSL
             -c 'select * from sys.nodes' > output.json
 
 
+=====================
+Environment Variables
+=====================
+
+Crash takes the following environment variables into account to select default
+connection parameters. These can be helpful to avoid user-input prompts or any
+other hard-coded cli parameter.
+
+:CRATEPWD: Password to be used if CrateDB demands password authentication.
+           For security reasons the usage of this variable is not recommended.
```

Comment:
We introduce a new feature but we discourage people from using it? I don't think that is an issue if the environment variable is not exposed.

Reply:
Agree, however we should also state that storing passwords in an env variable is not safe ;)